npm run build fails

## Describe the issue

when running a `npm run build` it fails with the following
```
PS E:\temp\SimpleWebAuthn\example> npm run build
> example@1.0.0 build
> tsc
fido-conformance.ts:143:8 - error TS2339: Property 'currentChallenge' does not exist on type 'LoggedInFIDOUser'.
143   user.currentChallenge = opts.challenge;
           ~~~~~~~~~~~~~~~~
fido-conformance.ts:160:34 - error TS2339: Property 'currentChallenge' does not exist on type 'LoggedInFIDOUser'.
160   const expectedChallenge = user.currentChallenge;
                                     ~~~~~~~~~~~~~~~~
fido-conformance.ts:225:8 - error TS2339: Property 'currentChallenge' does not exist on type 'LoggedInFIDOUser'.
225   user.currentChallenge = opts.challenge;
           ~~~~~~~~~~~~~~~~
fido-conformance.ts:242:34 - error TS2339: Property 'currentChallenge' does not exist on type 'LoggedInFIDOUser'.
242   const expectedChallenge = user.currentChallenge;
                                     ~~~~~~~~~~~~~~~~
Found 4 errors in the same file, starting at: fido-conformance.ts:143
```

## Reproduction Steps

1. `git clone https://github.com/MasterKale/SimpleWebAuthn.git`
2. `cd SimpleWebAuthn`
3.  `npm install`
4.  `npm run build`

## Expected behavior

Should build without an error

## Code Samples + WebAuthn Options and Responses

Already supplied

## Dependencies

- **OS:** Windows 11
- **Browser:** N/A
- **Authenticator:**N/A

### SimpleWebAuthn Libraries

<!-- Run the following command and paste in the output: -->

```sh
+-- @simplewebauthn/server@7.2.0
```

## Additional context

N/A

--- a/example/fido-conformance.ts
+++ b/example/fido-conformance.ts
@@ -144,6 +144,9 @@ fidoConformanceRouter.post('/attestation/options', (req, res) => {
 
   req.session.currentChallenge = opts.challenge;
 
+  // Only return the extensions we're given
+  opts.extensions = extensions;
+
   return res.send({
     ...opts,
     status: 'ok',

--- a/example/fido-conformance.ts
+++ b/example/fido-conformance.ts
@@ -140,7 +140,7 @@ fidoConformanceRouter.post('/attestation/options', (req, res) => {
     supportedAlgorithmIDs,
   });
 
-  user.currentChallenge = opts.challenge;
+  req.session.currentChallenge = opts.challenge;
 
   return res.send({
     ...opts,
@@ -157,7 +157,7 @@ fidoConformanceRouter.post('/attestation/result', async (req, res) => {
 
   const user = inMemoryUserDeviceDB[`${loggedInUsername}`];
 
-  const expectedChallenge = user.currentChallenge;
+  const expectedChallenge = req.session.currentChallenge;
 
   let verification;
   try {
@@ -222,7 +222,7 @@ fidoConformanceRouter.post('/assertion/options', (req, res) => {
     })),
   });
 
-  user.currentChallenge = opts.challenge;
+  req.session.currentChallenge = opts.challenge;
   user.currentAuthenticationUserVerification = userVerification;
 
   return res.send({
@@ -239,7 +239,7 @@ fidoConformanceRouter.post('/assertion/result', async (req, res) => {
   const user = inMemoryUserDeviceDB[`${loggedInUsername}`];
 
   // Pull up values specified when generation authentication options
-  const expectedChallenge = user.currentChallenge;
+  const expectedChallenge = req.session.currentChallenge;
   const userVerification = user.currentAuthenticationUserVerification;
 
   if (!id) {

--- a/example/fido-conformance.ts
+++ b/example/fido-conformance.ts
@@ -57,8 +57,10 @@ try {
 
 /**
  * Initialize MetadataService with Conformance Testing-specific statements.
+ *
+ * (Grabbed this URL from the POST made on https://mds3.fido.tools/ when you submit your site's URL)
  */
-fetch('https://mds3.certinfra.fidoalliance.org/getEndpoints', {
+fetch('https://mds3.fido.tools/getEndpoints', {
   method: 'POST',
   body: JSON.stringify({ endpoint: `${expectedOrigin}${fidoRouteSuffix}` }),
   headers: { 'Content-Type': 'application/json' },

